docs/documentation/server_admin/topics/organizations/managing-organization.adoc

```diff
@@ -0,0 +1,38 @@
+[id="managing-organization_{context}"]
+
+=  Managing an Organization
+[role="_abstract"]
+Organizations can be managed once you enable the `Organizations` setting in the `Realm Settings` page.
+
+Once you enable this setting, the `Organizations` section is available from the left side menu.
+
+By clicking on the `Organizations` section you can start creating an organization by clicking on the `Create Organization`
+button. If organizations already exist, you will see a list with all of them and options to search, edit, or delete an organization.
+
+An organization has the following settings:
+
+Name::
+A user-friendly name for the organization.
+
+Domains::
+A set of one or more domains that belongs to this organization. The domain is mainly used to map users to an organization
+based on their email domains as well as unlock different behaviors when managing and authenticating the organization members.
+
+Description::
+A free-text field to describe the organization.
+
+Once you create an organization, you are allowed to manage additional settings we will be looking at in the following sections:
+
+* Manage Attributes
+* Manage Members
+* Manage IdentityProviders
+
+==  Disabling an Organization
+
+You can disable an organization by clicking on the `Enabled` switch at the top of the page.
```

Comment:
```suggestion
To disable an organization toggle *Enabled*.
```

Reply:
Is there another way to indicate where *Enabled* is, such as next to a certain field?

Reply:
Yes, I can improve that.